docs/source/en/model_doc/dpt.mdx

```diff
@@ -34,6 +34,7 @@ A list of official Hugging Face and community (indicated by 🌎) resources to h
 
 - Demo notebooks for [`DPTForDepthEstimation`] can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DPT).
 - See also: [Semantic segmentation task guide](./tasks/semantic_segmentation)
+- [Monocular depth estimation task guide](./tasks/monocular_depth_estimation.mdx)
```

Comment:
ultra ultra nit: I would remove the `See also: ` in the line above to keep it consistent

Reply:
Done! :)